src/tests/ReactA11yInputElementsRuleTests.ts

```diff
@@ -20,10 +20,21 @@ describe('reactA11yInputElementsRule', (): void => {
         TestHelper.assertViolations(ruleName, script, []);
     });
 
+    it('should pass on input elements without placeholder of type radio, checkbox, file', (): void => {
+        const script: string = `
+            import React = require('react');
```

Comment:
Could you please also add tests for the case of inputs with attribute names other than `type`? Tests are passing without that check in the rule, so they're missing that case.

Reply:
Will do!